api/enki.py

```diff
@@ -557,16 +557,16 @@ class BibliographicParser(object):
     # Convert the English names of languages given in the Enki API to
     # the codes we use internally.
     LANGUAGE_CODES = {
-        "English": u"eng",
-        "French" : u"fre",
-        "Spanish": u"spa",
+        "English": "eng",
+        "French" : "fre",
+        "Spanish": "spa",
     }
 
     def process_all(self, json_data):
-        if isinstance(json_data, basestring):
+        if isinstance(json_data, (bytes, str)):
```

Comment:
I'm fine with this one going forward because json.loads works on both bytes and strings.